### PR TITLE
Add guide for db cluster creation with Terraform

### PR DESCRIPTION
**Pull request by Galiley**

### CODE REVIEW

File: pages/platform/databases/databases_09_order_terraform/guide.de-de.md [hunk 1]
```diff
@@ -243,9 +242,17 @@ export URI=$(terraform output -raw cluster_uri)
 
 And, voilà, the MongoDB cluster is created.
 
-## Go further
+## How to with other databases?
```
Comment (OVH-Team-Guides) [inline pages/platform/databases/databases_09_order_terraform/guide.de-de.md]: Isn't a verb missing here ?

File: pages/platform/databases/databases_09_order_terraform/guide.de-de.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+- Initialise backend (if defined)
+- Download and install modules (if defined)
+
+Now, we can generate our plan:
+
+```bash
+$ terraform plan -var-file=secrets.tfvars
+```
+
+Thanks to the `plan` command, we can check what Terraform wants to create, modify or remove.
```
Comment (BellionBastien) [inline pages/platform/databases/databases_09_order_terraform/guide.de-de.md]: same as init can you add a link to plan documentation

File: pages/platform/databases/databases_09_order_terraform/guide.de-de.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+The plan is OK for us, so let's apply it:
```
Comment (BellionBastien) [inline pages/platform/databases/databases_09_order_terraform/guide.de-de.md]: same as init can you add a link to apply documentation

File: pages/platform/databases/databases_09_order_terraform/guide.de-de.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+export URI=$(terraform output -raw cluster_uri)
+```
+
+And, voilà, the MongoDB cluster is created.
+
+## How to with other databases?
+
+In this guide, we explained how to deploy a MongoDB service but you can find example for other database engine here and tweak them according to your needs :
+
+* [MySQL](https://github.com/ovh/public-cloud-databases-examples/tree/main/databases/mysql/terraform/hello-world)
```
Comment (BellionBastien) [inline pages/platform/databases/databases_09_order_terraform/guide.de-de.md]: Here use the tab component: https://github.com/ovh/docs/blob/develop/pages/platform/databases/databases_03_advanced_configuration/guide.en-asia.md#get-the-desired-cluster-id

File: pages/platform/databases/databases_09_order_terraform/guide.de-de.md (hunk 1 above)
Comment (Galiley) [reply pages/platform/databases/databases_09_order_terraform/guide.de-de.md]: In that particular turn of phrase, the verb is redundant

**Review by BellionBastien — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @Galiley

**Pull request merged**